src/api/buddies.ts

```diff
@@ -78,27 +108,13 @@ export function banBuddy(
     );
 }
 
-const deleteRequest = (
-  buddyIds: string[],
-  accessToken: authApi.AccessToken,
-) => {
-  const buddyString = buddyIds.join(',');
-  const params = `?contact_user_ids=${buddyString}`;
-
-  return http.destroy(
-    `${config.baseUrl}/users/${accessToken.userId}/contacts${params}`,
-    {
-      headers: authApi.authHeader(accessToken),
-    },
-  );
-};
-
-export function deleteBuddies(
+export function banBuddies(
   buddyIds: string[],
+  banStatus: BanActions,
 ): (accessToken: authApi.AccessToken) => TE.TaskEither<string, Buddy> {
   return accessToken =>
     http.validateResponse(
-      deleteRequest(buddyIds, accessToken),
+      batchBanRequest(buddyIds, accessToken, banStatus),
       buddyType,
       toBuddy,
```

Comment:
These still worry me. toBuddy propably doesnt know what to do with the response from PATCH that atm returns multiple contacts eg: {resources: [{id: ...},{id: ...},]}

the PUT request i think response with {id: ...}

it is strange. the app doenst crash and i dont see anything weird in logs. but i dont understand how it could work.

Reply:
Fixed this